### ISSUE TITLE
FV3-JEDI executables failing due to missing library

### ISSUE THREAD

**Issue by SamuelDegelia-NOAA** [NOAA-EMC/RDASApp#133]
I am getting an error related to missing libraries when running any of the fv3jedi executables on Hera (`fv3jedi_var.x`, `fv3jedi_letkf.x`): 

```
34: /scratch1/NCEPDEV/da/Samuel.Degelia/RDASApp_dev/RDASApp/build/bin/fv3jedi_letkf.x: error while loading shared libraries: libfv3.so: cannot open shared object file: No such file or directory
```

This lib is built under `RDASApp/build/fv3/libfv3.so` so I am no sure what the problem is. Has anyone else experienced this since building with the updated submodules?

**Comment by guoqing-noaa** [comment]
I got the same error on Hera

**Comment by guoqing-noaa** [comment]
This is solved by PR #142
